pkg/apis/kops/registry/registry.go

```diff
@@ -29,6 +29,8 @@ const (
 	PathCluster = "config"
 	// Path for completed cluster spec in the state store
 	PathClusterCompleted = "cluster.spec"
+	// PathKopsVersion is the path for the version of kops last used to apply the cluster.
+	PathKopsVersionApplied = "kops-version-applied.txt"
```

Comment:
I think this would be closer to `cluster.spec`, `master-a`, `config` that are in the bucket root.
```suggestion
	PathKopsVersionApplied = "kops.version"
```

Reply:
Having the well-known `.txt` extension helps with troubleshooting.